crates/sui-framework/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+This file contains useful information and troubleshooting advice for those wishing to contribute to `sui-framework` crate.
+
+## Framework Move  source code changes
+
+If changes need to be made to the framework's Move code, additional actions need to be taken to ensure that the system builds and runs correctly. In particular, one needs to make sure that the framework snapshot tests are up-to-date and that any new native functions are correctly handled by the [Move Prover](https://github.com/move-language/move/tree/main/language/move-prover).
+
+### Snapshot tests update
+
+Run the following command in [crates/sui-cost](../sui-cost) and in [crates/sui-config](../sui-config) directories and accept the changes (if any):
+
+``` bash
+cargo insta test --review
```

Comment:
I think we can give a copy-paste tip like this:
```
cargo insta test -p sui-config -p sui-cost --review
```

Reply:
Another thing is that we need to give a tip on how cargo-insta is installed:
```bash
cargo install cargo-insta
```

Reply:
> Another thing is that we need to give a tip on how cargo-insta is installed:
> 
> ```shell
> cargo install cargo-insta
> ```

Good point! Added!

Reply:
> I think we can give a copy-paste tip like this (running all tests is unnecessary for this):
> 
> ```
> cargo insta test -p sui-config -p sui-cost --review
> ```

This does not seem to work and I did add a remark of running this command only in specific directories rather than in the Sui root dir.

Reply:
> This does not seem to work and I did add a remark of running this command only in specific directories rather than in the Sui root dir.

Whoopsie. Sorry for that; insta can only run one package at once. The replacement would be:
```
cargo insta test -p sui-cost --review
cargo insta test -p sui-config --review
```

I find it easier to run from a root. But maybe some folks prefer cd-ing.

Reply:
> > This does not seem to work and I did add a remark of running this command only in specific directories rather than in the Sui root dir.
> 
> Whoopsie. Sorry for that; insta can only run one package at once (unlike `cargo test`). The replacement would be:
> 
> ```
> cargo insta test -p sui-cost --review
> cargo insta test -p sui-config --review
> ```
> 
> I find it easier to run from the root. But maybe some folks prefer cd-ing.

I have no strong preference so changed it to match yours :-)